edges_estimate/plots.py

```diff
@@ -4,6 +4,13 @@
 from tqdm import tqdm
 from yabf import load_likelihood_from_yaml, LikelihoodContainer
 
+def get_evidence(mcsamplesr):
+    with open(mcsamplesr+".stats") as f:
```

Comment:
One thing: add a docstring that tells the user what `mcsamplesr` is: something like

```python
"""
Read the Bayesian evidence of a polychord run.

Parameters
----------
mcsamplesr : str
    The root file path of the run (without extension).
"""
```